Description: Make upstream buildsystem smarter with Debian policy.
 Make the Makefile behave on a smarter way within Debian policy requirements (4.9.1).
 * Don't strip the binary here (done by dh_strip)
 * s/CCFLAGS/CFLAGS/g : Let dpkg-buildpackage define CFLAGS.
Origin: other
Author: Didier Raboud <didier@raboud.com>
Last-Update: 2009-07-02

Index: usb-modeswitch.git/Makefile
===================================================================
--- usb-modeswitch.git.orig/Makefile	2009-08-23 21:08:20.000000000 +0200
+++ usb-modeswitch.git/Makefile	2009-08-23 21:09:38.000000000 +0200
@@ -2,7 +2,7 @@
 VERS        = 1.0.3
 STRIP	    = strip
 CC          = gcc
-CCFLAGS     = -l usb
+CFLAGS     += -l usb
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
 PREFIX	    = $(DESTDIR)/usr
@@ -12,8 +12,7 @@
 .PHONY:     clean
 all:        $(PROG)
 $(PROG):    $(OBJS)
-	      $(CC) $(CCFLAGS) -o $(PROG) $(OBJS)
-	      $(STRIP) $(PROG)
+	      $(CC) $(CFLAGS) -o $(PROG) $(OBJS)
 
 clean:
 	      $(RM) ./usb_modeswitch
Index: usb-modeswitch.git/Makefile.integrated
===================================================================
--- usb-modeswitch.git.orig/Makefile.integrated	2009-08-23 21:09:40.000000000 +0200
+++ usb-modeswitch.git/Makefile.integrated	2009-08-23 21:09:50.000000000 +0200
@@ -2,7 +2,7 @@
 VERS        = 1.0.3
 STRIP       = strip
 CC          = gcc
-CCFLAGS     = -l usb -DUSE_SYSLOG
+CFLAGS     += -l usb -DUSE_SYSLOG
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
 PREFIX      = $(DESTDIR)/usr
@@ -14,8 +14,7 @@
 all:        clean $(PROG)
 
 $(PROG): $(OBJS)
-	$(CC) $(CCFLAGS) -o $(PROG) $(OBJS)
-	$(STRIP) $(PROG)
+	$(CC) $(CFLAGS) -o $(PROG) $(OBJS)
 
 clean:
 	$(RM) ./usb_modeswitch
